basyx/aas/adapter/http.py

```diff
@@ -591,21 +605,38 @@ def _get_shells(self, request: Request) -> Iterator[model.AssetAdministrationShe
     def _get_shell(self, url_args: Dict) -> model.AssetAdministrationShell:
         return self._get_obj_ts(url_args["aas_id"], model.AssetAdministrationShell)
 
-    def _get_submodels(self, request: Request) -> Iterator[model.Submodel]:
-        submodels: Iterator[model.Submodel] = self._get_all_obj_of_type(model.Submodel)
+    def _get_submodels(self, request: Request) -> Tuple[Iterator[model.Submodel], int]:
+        limit = request.args.get('limit', type=int, default=10)
```

Comment:
Can you move the slicing to a separate function, so parameter names and default values such as `limit`, `cursor`, `10`, etc. are only specified once?